src/shared/combineCommandSequences.ts

```diff
@@ -77,35 +77,3 @@ export function combineCommandSequences(messages: ClineMessage[]): ClineMessage[
 			return msg
 		})
 }
-
-export const splitCommandOutput = (text: string) => {
```

Comment:
This was expensive to run when there's lot of terminal output. I don't think it provided enough value to justify the performance hit.